platform/windows/export/export_plugin.cpp

```diff
@@ -327,3 +328,46 @@ Error EditorExportPlatformWindows::_code_sign(const Ref<EditorExportPreset> &p_p
 
 	return OK;
 }
+
+bool EditorExportPlatformWindows::can_export(const Ref<EditorExportPreset> &p_preset, String &r_error, bool &r_missing_templates) const {
+	String err = "";
+	bool valid = EditorExportPlatformPC::can_export(p_preset, err, r_missing_templates);
+
+	String rcedit_path = EditorSettings::get_singleton()->get("export/windows/rcedit");
+	if (rcedit_path.is_empty()) {
+		err += TTR("The rcedit tool must be configured in the Editor Settings (Export > Windows > Rcedit) to change the icon or app information data.") + "\n";
+	}
+
+	String icon_path = ProjectSettings::get_singleton()->globalize_path(p_preset->get("application/icon"));
+	if (!icon_path.is_empty() && !FileAccess::exists(icon_path)) {
+		err += TTR("Invalid icon path: ") + icon_path + "\n";
```

Comment:
Trailing whitespace is difficult to translate, it's better to keep it out:
```suggestion
		err += TTR("Invalid icon path:") + " " + icon_path + "\n";
```